module/learningResources/helper.js

```diff
@@ -0,0 +1,265 @@
+/** 
+* name : helper.js
+* author : Rakesh Kumar
+* created-date : 24-Jun-2020
+* Description : Related to learning resources
+*/
+
+const sunbirdService =
+  require(GENERIC_SERVICES_PATH + "/sunbird");
+
+/**
+* Learning resource related information be here.
+* @method
+* @class  LearningResourcesHelper
+*/
+
+module.exports = class LearningResourcesHelper {
+
+  /**
+  * To get list of learning resources
+  * @method
+  * @name  all
+  * @param {String} token - user access token.
+  * @param {String} pageSize - page size of the request
+  * @param {String} pageNo - page no of the request
+  * @param {Object} filters - resource filters
+  * @param {Array} filters.category - categories for the learning resource
+  * @param {Array} filters.subCategory - subcategories for the learning resource
+  * @param {Array} filters.topic - topic's for the learning resource
+  * @param {Array} filters.language - language's of the learning resources
+
+  * @returns {json} Response consists of list of learning resources
+  */
+  static all(token, pageSize, pageNo, filters) {
+    return new Promise(async (resolve, reject) => {
+      try {
+
+        let popularResources = await this.popular(token, pageSize, pageNo, filters);
+        let recentResources = await this.recentlyAdded(token, pageSize, pageNo, filters);
+        let allResources = [];
+        if (recentResources && recentResources.data) {
+          allResources.push(recentResources.data);
+        }
+        if (popularResources && popularResources.data) {
+          allResources.push(popularResources.data);
+        }
+        if (allResources && allResources.length > 0) {
+          resolve({
+            message: CONSTANTS.apiResponses.LEARNING_RESORCES_FOUND,
+            data: allResources,
+            success: true
+          });
+        } else {
+          resolve({
+            message: popularResources.message,
```

Comment:
@rakeshSgr - Make it a common message combining both and make it as a throw error statement.